app/routes/goal_routes.py

```diff
@@ -1 +1,108 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, request, Response
+from ..models.goal import Goal
+from ..models.task import Task
+from ..db import db
+
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
```

Comment:
Prefer blueprint to be named `bp`.

```suggestion
bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
```